log/log.go

```diff
@@ -0,0 +1,63 @@
+package log
```

Comment:
since this is just the definition of what a logger is... and a utility function to create one... We could have it in `runtime/log.go` it's not super important...